javascript/javascript1/week3/index.js

```diff
@@ -0,0 +1,205 @@
+}
+console.log(`the total hours to travel is ${travelTime(hours)}`);
+
+
+// SERIES DURATION MY LIFE
+
+const seriesDurations = {
+    "first":{
+        title: 'Game of thrones',
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    },
+    "second":{
+        title: 'Sopranos',
+        days: 3,
+        hours: 14,
+        minutes: 0,
+
+    },
+    "third":{
+        title: 'The Wire',
+        days: 2,
+        hours: 12,
+        minutes: 0,
+    }
+};
+const myLifeSpan = 80*365*24;
+let totalTimeForFirstSeries =(seriesDurations.first["days"]* (seriesDurations.first["hours"]+seriesDurations.first["minutes"]/60));
+let totalTimeForSecondSeries =(seriesDurations.first["days"]* (seriesDurations.second["hours"]+seriesDurations.second["minutes"]/60));
+let totalTimeForThirdSeries =(seriesDurations.first["days"]* (seriesDurations.third["hours"]+seriesDurations.third["minutes"]/60));
+//console.log(totalTimeForFirstSeries);
+//console.log(totalTimeForSecondSeries)
+//console.log(totalTimeForThirdSeries)
+let inTotalTimeForSeries = totalTimeForFirstSeries + totalTimeForSecondSeries + totalTimeForThirdSeries;
+
+// Function to calculate percentage
+function logOutSeriesText(time,year){
+    percentage = time/year *100;
+    return percentage;
+
+}
+const firstSeries = logOutSeriesText(totalTimeForFirstSeries, myLifeSpan);
+const secondSeries = logOutSeriesText(totalTimeForSecondSeries, myLifeSpan);
+const thirdSeries = logOutSeriesText(totalTimeForThirdSeries, myLifeSpan);
+const inTotal = logOutSeriesText(inTotalTimeForSeries, myLifeSpan)
+console.log(`Game of thrones took${firstSeries}% of my life`);
+console.log(`Sopranos took${secondSeries}% of my life`);
+console.log(`The Wire took${thirdSeries}% of my life`);
+console.log(`In total that is${inTotal}% of my life`);
+
+// Step3: Note taking app!
+
+//3.1 save a note
+const notes = [];
+function saveNote(content, id){
+    let note = {
+        content: content,
+        id: id
+    };
+    return notes.push(note);
+}
+saveNote("Pickup groceries", 1);
+saveNote(" Do laundry", 2);
+console.log(notes);
+
+// 3.2 Get a note
+
+function getNote(id){
+    for(let i = 0; i<notes.length; i++){
+        if(id === isNaN() ||id ==='' ){
+            return  'please write a number in ID'
+        }
+        else if(id === (notes[i].id)){
+            return notes[i];
+        }
+        
+    }
+
+
+}
+const firstNote = getNote(1);
+console.log(firstNote);
+
+
+//3.3 log out notes
+
+function logOutNotesFormatted(){
+    for(i=0; i<notes.length; i++){
+        console.log(`The note with id: ${notes[i].id}, has the following note text: ${notes[i].content}`);
+        
+    }
+}
+logOutNotesFormatted();
+
+// Adding a activity and Usage Limit
+
+const activities = [];
+
+function addActivity(date, activity, duration){
+    let addedActivities = {date: new Date(), activity: activity, duration: duration};
+    if (typeof activity === "string" && !isNaN(duration && activity!="")){
```

Comment:
It looks like you're passing `duration && activity != ""` to `isNaN`. Remember that `isNaN` only checks if a number is valid so it should only accept `duration`